README.md

```diff
@@ -62,6 +62,7 @@ mvn clean install
 | OKAPI_URL                                       | http://okapi:9130        | Okapi URL, used for system user authentication/management                                                   |
 | SYSTEM_USER_PASSWORD                            | -                        | Password for the system user; **must be set**                                                               |
 | SYSTEM_USER_ENABLED                             | true                     | Defines if system user must be created at service tenant initialization or used for egress service requests |
+| SYSTEM_USER_RETRY_WAIT_MINUTES                  | 10                       | Max time to wait for the system user to be created, which is used in the Tenant API                         |
```

Comment:
please add this to the module descriptor as well

Reply:
thanks, updated